src/models/data.cpp

```diff
@@ -133,6 +133,147 @@ ItemCost buildCallerCalleeResult(const BottomUp& data, const Costs& bottomUpCost
     }
     return totalCost;
 }
+
+bool startsWith(const QStringRef& str, const QLatin1String& prefix, int& size)
```

Comment:
please return a negative size to report a mismatch and don't take the size as an out param

Reply:
Would it be acceptable if I changed end to be a pointer? Returning it would make the calling code much uglier.